src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IPropertyReferenceExpression.vb

```diff
@@ -70,5 +70,64 @@ IPropertyReferenceOperation: Property M1.C1.P1 As System.Object (OperationKind.P
 
             VerifyOperationTreeAndDiagnosticsForTest(Of MemberAccessExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub IPropertyReference_SharedPropertyWithInstanceReceiver()
```

Comment:
>IPropertyReference_SharedPropertyWithInstanceReceiver [](start = 19, length = 53)

Please make sure we have tests for accessing a property (static and instance) of off a type

Reply:
Done.

---
In reply to: [149249549](https://github.com/dotnet/roslyn/pull/22972#discussion_r149249549) [](ancestors = 149249549)